### PR TITLE
Add cppcheck

### PR DESCRIPTION
**Pull request by RicardoMDomingues**

### CODE REVIEW

File: bridge/engine/EngineAudioStream.h [hunk 1]
```diff
@@ -18,26 +18,26 @@ struct EngineAudioStream
...
-        bool ssrcRewrite,
-        const uint32_t idleTimeoutSeconds,
+    EngineAudioStream(const std::string& endpointId_,
+        const size_t endpointIdHash_,
+        const uint32_t localSsrc_,
+        const utils::Optional<uint32_t>& remoteSsrc_,
+        transport::RtcTransport& transport_,
+        const bool audioMixed_,
+        const bridge::RtpMap& rtpMap_,
+        bool ssrcRewrite_,
```
Comment (olofkallander) [inline bridge/engine/EngineAudioStream.h:28]: Did cpp check complain about this. Iirc the C++ standard resolves exactly these situations without postfixing etc.

Comment (RicardoMDomingues) [reply bridge/engine/EngineAudioStream.h:28]: Yeah, unfortunately it did

File: transport/ProbeServer.cpp [hunk 2]
```diff
@@ -304,12 +304,16 @@ void ProbeServer::run()
...
         auto now = utils::Time::getAbsoluteTime();
 
-        for (auto it = _tcpConnections.begin(); it != _tcpConnections.end(); ++it)
+        for (auto it = _tcpConnections.begin(); it != _tcpConnections.end(); )
         {
             if (utils::Time::diffGT(it->timestamp, now, _iceConfig.probeConnectionExpirationTimeout * utils::Time::sec))
             {
                 it->endpoint->stop(this);
-                _tcpConnections.erase(it);
+                it = _tcpConnections.erase(it);
```
Comment (RicardoMDomingues) [inline transport/ProbeServer.cpp:312]: This was a nice catch that cppcheck did 😃

File: bridge/VideoStream.h [hunk 3]
```diff
@@ -14,24 +14,24 @@ namespace bridge
...
+        const std::string& endpointId_,
+        const uint32_t localSsrc_,
+        std::shared_ptr<transport::RtcTransport>& transport_,
+        bool ssrcRewrite_,
+        bool isDtlsLocalEnabled_,
         utils::Optional<uint32_t> idleTimeout)
-        : id(id),
-          endpointId(endpointId),
+        : id(id_),
+          endpointId(endpointId_),
```
Comment (olofkallander) [inline bridge/VideoStream.h:25]: We do not need the _. Why what is requiring it?

File: bridge/engine/EngineAudioStream.h (hunk 1 above)
Comment (olofkallander) [reply bridge/engine/EngineAudioStream.h:28]: Can we suppress it because it is not a real error?

**Review by reddvl1980 — APPROVED** [review]

**Pull request merged**